USAGE.md

```diff
@@ -38,7 +38,7 @@ print(header.json_string())
 * [Categories](#categories)
 * [Schedulling Parameters](#schedulling-parameters)
```

Comment:
Typo? #88 also fixes this typo but only the second occurrence on line 155. With one or two extra commits, this PR could make #88 redundant.

Reply:
Thanks, I'll suggest the fixes in #88.